src/harness/reference_models/pre_iap_filtering/fss_purge.py

```diff
@@ -0,0 +1,301 @@
+#    Copyright 2018 SAS Project Authors. All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+"""
+==================================================================================
+    This is a subset of the Pre-IAP reference model which implements
+    the FSS purge list algorithm described in R2-SGN-29.
+    The main function is fssPurgeModel().
+==================================================================================
+"""
+import numpy as np
+from collections import namedtuple
+from reference_models.antenna import antenna
+from reference_models.interference import interference
+from reference_models.geo import vincenty
+from reference_models.propagation import wf_itm
+from reference_models.pre_iap_filtering import pre_iap_util
+
+# Definition of constants and data types for FSS purge model.
+# Reference Channel bandwidth for FSS OOBE calculation
+REF_BW = 5.e6
+
+# OOBE Margin
+FSS_OOBE_MARGIN = 1
```

Comment:
Shouldn't the margin be set to "2" instead of "1"? Document WINNF-SSC-007-V1.0.0 states the following:

In release 1 of WinnForum specifications, the following values are agreed among and used by all SAS Administrators.
Mg1 = Mg2 = Mg3 = Mg4 = Mg5 = Mg6= 2dB.

Reply:
The newer Document WINNF-SSC-007-V1.0.1 states
Mg1 = Mg2 = Mg3 = Mg4 = Mg5 = Mg6= 1dB.